src/Compilers/CSharp/Portable/Errors/LazyUnmanagedCallersOnlyMethodCalledDiagnosticInfo.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Diagnostics;
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal sealed class LazyUnmanagedCallersOnlyMethodCalledDiagnosticInfo : DiagnosticInfo
+    {
+        private DiagnosticInfo? _lazyActualUnmanagedCallersOnlyDiagnostic;
+
+        private readonly MethodSymbol _method;
+
+        internal LazyUnmanagedCallersOnlyMethodCalledDiagnosticInfo(MethodSymbol method)
+            : base(CSharp.MessageProvider.Instance, (int)ErrorCode.Unknown)
```

Comment:
Why do we use the Unknown error code here?

Reply:
Because that's what `Obsolete` does.

---
In reply to: [481490517](https://github.com/dotnet/roslyn/pull/47311#discussion_r481490517) [](ancestors = 481490517)